events/manager/manager.go

```diff
@@ -31,6 +31,9 @@ var (
 			Name:      "last_time_seconds",
 			Help:      "Last time of eventer housekeep since unix epoch in seconds.",
 		})
+
+	// Time of latest scrape operation
+	LatestScrapeTime = time.Now()
```

Comment:
I am afraid yes since `manager` is initialized in `main` package. We can not get the same manager instance.